chatbot/model_service/builds/x86-cuda/Containerfile

```diff
@@ -1,10 +1,14 @@
 FROM quay.io/opendatahub/workbench-images:cuda-ubi9-python-3.9-20231206
 WORKDIR /locallm
-COPY builds/requirements.txt /locallm/requirements.txt
+# MODEL_FILE should be set at runtime
+ENV MODEL_FILE=llama-2-7b-chat.Q5_K_S.gguf
+ENV MODEL_PATH=/locallm/models/${MODEL_FILE}
+COPY chatbot/model_service/builds/requirements.txt .
```

Comment:
I'm not convinced that we can run `podman build` from the top directory without a major lag due to copying the entire repo, specifically the models dir. Is there any other way around this? Maybe push all other code down one level and the top dir is just like "src" and "models" the we build everything from "src" dir?

Reply:
we can do that, I'll move things around

Reply:
I updated - only the chatbot & we can update the summarizer in a followup - 
```
chatbot/model_services/base
chatbot/model_services/cuda
chatbot/model_services/src
```
and the shared files live in src (requirements.txt, chat_service.py, llamacpp_utils.py)